DESCRIPTION

```diff
@@ -70,6 +70,6 @@ Config/Needs/website: tidyverse/tidytemplate
 Config/testthat/edition: 3
 Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.3.1
-Remotes:  
+RoxygenNote: 7.3.1.9000
```

Comment:
I generally have feelings on using dev roxygen since the version number doesn't map to a definite SHA. Any reason this is required here?

Reply:
mistake on my part